packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -88,6 +101,24 @@ export default class QuanticSearchBox extends LightningElement {
         },
       },
     });
+
+    if (
+      !this.disableRecentQuerySuggestions &&
+      this.headless.buildRecentQueriesList
+    ) {
+      this.localStorageKey = `${this.engineId}_quantic-recent-queries`;
+      this.recentQueriesList = this.headless.buildRecentQueriesList(engine, {
+        initialState: {
+          queries: getItemFromLocalStorage(this.localStorageKey) ?? [],
+        },
+        options: {
+          maxLength: 100,
```

Comment:
the default in the Headless recent queries is 10 but this does't suit this current use case.
Because we are not just displaying recent queries, but we are displaying recent queries based on the query written in the search box which means we need a larger number of recent queries cause a lot of them will be filtered out based on the current query.

the choice of a 100 is just a large number.